src/components/main.js

```diff
@@ -224,6 +243,7 @@ export default class MainPage extends Component {
                     <hr className={style.fullWidthHr}></hr>
                     <div className={style.textAlignCenter}>
                     <span className={classnames(style.value, style.viewPort)} onClick = {() => {window.open(`https://etherscan.io/address/${TOKEN_CONTRACT}`,  '_blank');}}>{TOKEN_CONTRACT}</span>
+                    <span className={classnames(style.value, style.viewPortSmallPage)} onClick = {() => {window.open(`https://etherscan.io/address/${TOKEN_CONTRACT}`,  '_blank');}}>{format(TOKEN_CONTRACT)}</span>
```

Comment:
Only one of the two are displayed using media queries, which means on mobile you see full address and regular desktops but the very small desktops this would cause issue on like 1000 pixel screens I have the dots in the middle of the address